src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
The div element could be replaced with a more semantic HTML element. A section or article element may be a better choice here. It's a good practice to use semantic elements when possible as they provide additional information about the type of content contained within the element. Semantic elements can improve accessibility and SEO.